profiles/prefix/darwin/macos/14.0/arm64/clang/make.defaults

```diff
@@ -0,0 +1,4 @@
+LD=ld64.lld
```

Comment:
it feels like most of this stuff belongs in some generic place, e.g. under profiles/prefix/darwin/macos/features/clang

then of course link to it, such that macos 15, 16, etc. all can just link to this set of changes instead redefining them again, including of course support for x64, etc.

Reply:
I think done. I don't know whether ld64 is used on x64 though